front/src/pages/MainPage/components/Grid/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { Grid, CircularProgress } from "@material-ui/core";
+import { useDispatch, useSelector } from "react-redux";
+
+import {
+  addItemToBasket,
+  selectBooksFetchStatus,
+  selectBooksData,
+} from "features/books/booksSlice";
+
+import { Card } from "./components/index";
+
+interface Book {
```

Comment:
znowu to dublujesz, trzymaj jeden interface na to w jednym pliku i importuj

Reply:
Done.